plugins/usbpro/RobeWidgetDetector.h

```diff
@@ -56,6 +56,12 @@ class RobeWidgetInformation {
   uint8_t hardware_version;
   uint8_t software_version;
   uint8_t eeprom_version;
+  void operator=(const RobeWidgetInformation& other) {
+      hardware_version = other.hardware_version;
+      software_version = other.software_version;
+      eeprom_version = other.eeprom_version;
+      uid = other.uid;
```

Comment:
Nit, would it make sense for the order of these to match the constructor that passes in other? I'm not quite sure what your order is, apart from basic types first, but not even alphabetical.

Reply:
Done.